cypress/support/graphql-utils.ts

```
@@ -0,0 +1,147 @@
+};
+
+/**
+ * Alias a GraphQL operation for later waiting
+ * This sets up an intercept that assigns an alias when the operation matches
+ *
+ * @param operationName - The GraphQL operation name to alias
+ *
+ * @example
+ * cy.aliasGraphQLOperation('GetOrganizations');
+ * cy.visit('/orglist');
+ * cy.wait('@GetOrganizations');
```

ESLint

[error] tsdoc-at-sign-in-word: The "@" character looks like part of a TSDoc tag; use a backslash to escape it

(tsdoc/syntax)